src/sql/workbench/api/browser/mainThreadNotebook.ts

```diff
@@ -235,7 +235,9 @@ class SessionManagerWrapper implements azdata.nb.SessionManager {
 
 	private async doStartNew(options: azdata.nb.ISessionOptions): Promise<azdata.nb.ISession> {
 		let sessionDetails = await this._proxy.ext.$startNewSession(this.managerHandle, options);
-		return new SessionWrapper(this._proxy, sessionDetails);
+		const sessionManager = new SessionWrapper(this._proxy, sessionDetails);
+		await sessionManager.initialize();
```

Comment:
Where was initialize being called before this? Is doing it here going to break anything?